docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3074.md

```diff
@@ -0,0 +1,41 @@
+# GUI Launch form: disk size disclaimers
+
+Test verifies that 
+- special warning can be specified for job depending on the job disk size
+- pause action can be disabled for the job depending on the job disk size
+- if several configurations are active, the one with the largest threshold will be displayed
+
+
+**Prerequisites**:
+- Admin user
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Login as the admin user from the prerequisites | |
+| 2 | Open the **Settings** page | |
+| 3 | Open the **PREFERENCES** tab | |
+| 4 | Click the **Launch** tab | |
+| 5 | Find the **`launch.job.disk.size.thresholds`** preference | |
+| 6 | Save the preference value, then input follow json as preference value: <br>`[` <ul> `{` <ul> `"threshold": 30,` <br> `"disclaimer": "Threshold disclaimer 1.",` <br>  `"disclaimerType": "info"` </ul> `},` <br> `{` <ul> `"threshold": 50,` <br> `"disclaimer": "Threshold disclaimer 2. Your job size: {size}Gb, larger or equal {threshold}Gb",` <br>  `"pause": false` </ul> `},` <br> `{` <ul> `"threshold": 70,` <br> `"disclaimer": "Threshold disclaimer 3. Your disk: {disk}Gb, larger then {threshold}Gb",` <br>  `"disclaimerType": "error"` </ul> `}` </ul> ` ]` |
+| 7 | Click **Save** button | |
+| 8 | Open the **Tools** page | |
+| 9 | Select the test tool from the prerequisites | |
+| 10 | Click the ***Run with custom settings*** button | |
+| 11 | Expand the **Advanced** section | |
+| 12 | Select `On-demand` value for **Price type** | |
+| 13 | Expand **Exec environment** section | |
+| 14 | Input `40` into the **_Disk (Gb)_** field | |
+| 15 | Click **Launch** button | *Launch* pop up appears that contains Info message `Threshold disclaimer 1.` |
+| 16 | Click **Cancel** button | |
+| 17 | Input `60` into the **_Disk (Gb)_** field | |
+| 18 | Click **Launch** button | *Launch* pop up appears that contains Warning message `Threshold disclaimer 2. Your job size: 50Gb, larger then 50Gb` |
```

Comment:
Warning message will be `Threshold disclaimer 2. Your job size: 60Gb, larger or equal 50Gb`